app/src/index.css

```diff
@@ -1,4 +1,8 @@
 @import "tailwindcss/base";
 @import "@navikt/ds-css";
 @import "tailwindcss/components";
-@import "tailwindcss/utilities";
\ No newline at end of file
+@import "tailwindcss/utilities";
+
+hr {
+    color: var(--a-gray-300);
+}
```

Comment:
Lag heller en komponent `HorizontalRule` som basically er dette:

```tsx
export const HorizontalRule = () => {
  return <hr className="color-gray-300" />;
}
```

(eller hva enn det klassenavnet er). Da slipper vi globale stiler, som kan herpe for andre ting på siden om 2-3 år.